API_Guidelines.md

```diff
@@ -70,15 +70,16 @@ Payload
 ```
 
 ### GET multiple records using a query
-Calling store.query() will make a `GET` request with the passed object serialized as query params. This method returns a DS.PromiseArray in the same way as findAll.
-
-Further reference: [Finding Records](https://guides.emberjs.com/v2.10.0/models/finding-records/)
-
+#### Request
 URL
 :   `apiHost.com/movies?year=1980`
 
 Ember Data method
-:   `this.store.query('movie', { year: '1980' })`
+:   `query('movie', { year: '1980' })`
+
+### Why?
+
+> To get multiple records based on parameter criteria.
 
 ### Response
 
```

Comment:
minor change but if the year is 1980, goodfellas should not be the resposne, it should be 

```
    {
      "id": 1,
      "title": "Raging Bull",
      "year": "1980"
    }
```